stories/wizard/wizard.stories.js

```diff
@@ -0,0 +1,868 @@
+                <div class="fd-wizard__step-wrapper">
+                    <a class="fd-wizard__step-container" tabindex="0" aria-label="Step 3: Branching" aria-disabled="true">
+                        <span class="fd-wizard__step-indicator">3</span>
+                        <div class="fd-wizard__label-container fd-wizard__label-container--optional">
+                            <span class="fd-wizard__label">Step 3: Branching</span>
+                        </div>
+                    </a>
+                    <span class="fd-wizard__connector fd-wizard__connector--branching"></span>
+                </div>
+            </li>
+        </ul>
+    </nav>
+    <section class="fd-wizard__content fd-wizard__content--list fd-wizard__content--m" id="wizard-section-2" style="min-height: 300px;">
+        <div>
+            Sed fermentum, mi et tristique ullamcorper, sapien sapien faucibus sem, quis pretium nibh lorem malesuada diam. Nulla quis arcu aliquet, feugiat massa semper, volutpat diam. Nam vitae ante posuere, molestie neque sit amet, dapibus velit. Maecenas eleifend tempor lorem. Mauris vitae elementum mi, sed eleifend ligula. Nulla tempor vulputate dolor, nec dignissim quam convallis ut. Praesent vitae commodo felis, ut iaculis felis. Fusce quis eleifend sapien, eget facilisis nibh. Suspendisse est velit, scelerisque ut commodo eget, dignissim quis metus. Cras faucibus consequat gravida. Curabitur vitae quam felis. Phasellus ac leo eleifend, commodo tortor et, varius quam. Aliquam erat volutpat
+        </div>
+        <div class="fd-wizard__next-step">
+            <button class="fd-button fd-button--compact fd-button--emphasized">Step 3</button>
+        </div>
+    </section>
+    <footer>
+        <div class="fd-bar fd-bar--page-m_l fd-bar--footer">
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button class="fd-button fd-button--transparent fd-button--compact">Cancel</button>
+                </div>
+            </div>
+        </div>
+    </footer>
+</div>
+`;
+
+customized.parameters = {
+    docs: {
+        iframeHeight: 950,
+        storyDescription: `Modifier classes can be applied to customize the Wizard's elements. In the example below responsive paddings are added to the progress bar (<code class="docs-code">fd-wizard\\_\\_progress-bar--m</code>), the wizard content (<code class="docs-code">fd-wizard\\_\\_content--m</code>) and the footer (<code class="docs-code">fd-bar--page-m_l</code>). The background of the wizard content is set to list (<code class="docs-code">fd-wizard\\_\\_content--list</code>).
+
+In certain use cases, the steps in the wizard depend on the choices the user makes along the way. The user’s entries for one step determine the follow-on steps (“branching”). Use the <code class="docs-code">fd-wizard\\_\\_connector--branching</code> applied on the step connector to indicate a branching step.
+`
+    }
```

Comment:
The wizard component can be customized with modifier classes. For instance, the example below has several modified elements:

| Modified | Modification |
| :------------- | :------------|
| <code class="docs-code">fd-wizard\\_\\_progress-bar--m</code> | Added responsive padding |
| <code class="docs-code">fd-wizard\\_\\_content--m</code> | Added responsive padding |
| <code class="docs-code">fd-bar--page-m_l</code> | Added responsive padding |
| <code class="docs-code">fd-wizard\\_\\_content--list</code> | The background of the wizard content is set to list |

###Branching
To display a branching step, add the <code class="docs-code">fd-wizard\\_\\_connector--branching</code> to the step connector element.

Reply:
This I changed a bit. These are modifier classes, so I added a column for the element and a column for the modifier class